### ISSUE TITLE
Webpack 2.4.0 breaks compiling angular 1.5.11

### ISSUE THREAD

**Issue by shssoichiro** [webpack/webpack#4709]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
When building a project that pulls in Angular ~1.5 as a dependency, the following warnings are produced:

```
WARNING in ./~/angular/angular.js
7827:17-24 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

WARNING in ./~/angular/angular.js
7827:38-45 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

WARNING in ./~/angular/angular.js
7828:14-21 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

WARNING in ./~/angular/angular.js
7831:19-26 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

WARNING in ./~/angular/angular.js
7835:11-18 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
```

If the build is using `-p`, the following **error** is also produced:

```
ERROR in bundle.js from UglifyJs
Invalid assignment [./~/angular/angular.js:7831,0][bundle.js:62375,150]
```

**If the current behavior is a bug, please provide the steps to reproduce.**
1. Create a project with Angular 1.5.11 imported by one of the app's JS files.
2. Run webpack.

**What is the expected behavior?**
There should be no warnings or errors. Works correctly in Webpack 2.3.3.

**If this is a feature request, what is motivation or use case for changing the behavior?**
N/A

**Please mention other relevant information such as the browser version, Node.js version, webpack version and Operating System.**
Node.js 6.10.2, Webpack 2.4.0, tested on OS X 10.12.3 and Debian Wheezy.

**Comment by sokra** [comment]
I'll take a look

**Comment by sokra** [comment]
fixed, published as 2.4.1
